src/main/kotlin/me/ezra_home/retail_software_solution/rest/endpoints/sysuser/FruitEndpoint.kt

```diff
@@ -0,0 +1,78 @@
+package me.ezra_home.retail_software_solution.rest
+
+import me.ezra_home.retail_software_solution.businesses.sysuser.service.FruitService
+import me.ezra_home.retail_software_solution.model.entity.FruitEntity
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/fruits")
```

Comment:
- make this secured
- we don't need to prefix with `api`
- The opening slash is not needed